students/7/main.js

```diff
@@ -56,8 +56,27 @@ const getFirstName = async (userId) => {
 }
 
 const ex4 = async () => {
-    console.log("TODO...")
+    let names = await getNames();
+    console.log(names);
+}
+
+
+const getNames = async () => {
+    let userNames = [];
+        try {
+                
+            let result = await axios.get(`https://jsonplaceholder.typicode.com/users/`);
+                for (let i of result.data) {
```

Comment:
Can you fix the indentation?  This will be problematic when we start Python.

Reply:
Sure, indentation on which lines?